packages/node_modules/@ciscospark/react-component-activity-item/src/index.js

```diff
@@ -58,14 +90,20 @@ function ActivityItem(props) {
     );
   }
   else if (verb === SHARE_VERB) {
-    itemComponent = (
-      <ActivityShareFiles
-        content={activity.content}
-        displayName={activity.displayName}
-        files={activity.files.items}
-        {...props}
-      />
-    );
+    if (activity.contentCategory === LINKS_CONTENT_CATEGORY) {
+      // ECM files will be displayed as links for now
+      itemComponent = createECMItemComponents(props);
```

Comment:
It might be worthwhile to make this its own react component so that we can style it properly once we get UI info on how this is supposed to look. Not a blocker though.